src/pages/dashboard/index.vue

```diff
@@ -300,13 +300,18 @@ export default {
 
       this.isIncomeDetailsLoading = true;
       this.isOpenIncomeDetailsDialog = true;
-      if (!this.getTotalSales && !this.getTotalRoyalty) {
-        await Promise.all([
-          this.walletFetchTotalSales(this.wallet),
-          this.walletFetchTotalRoyalty(this.wallet),
-        ]);
+      try {
+        if (!this.getTotalSales && !this.getTotalRoyalty) {
+          await Promise.all([
+            this.walletFetchTotalSales(this.wallet),
+            this.walletFetchTotalRoyalty(this.wallet),
+          ]);
+        }
+      } catch (error) {
+        // no-op
```

Comment:
Please `console.error` if error is not expected
or just don't have a `catch` block if the error does not need to be caught